docs/src/domains/cdn/managed-fastly.md

```diff
@@ -0,0 +1,48 @@
+---
+title: "Managed Fastly CDN"
+sidebarTitle: "Managed Fastly CDN"
+weight: 2
+description: Bring your content closer to users with a Fastly CDN fully managed by Platform.sh.
+tier:
+  - Elite
+  - Enterprise
+---
+
+Instead of starting your own Fastly subscription and [managing your CDN yourself](./fastly.md),
+you can take advantage of a Fastly CDN provided by Platform.sh.
+For example, Dedicated projects include a managed Fastly CDN by default.
+These CDNs are exclusively set up and managed by Platform.sh.
+
+To modify any settings for a managed Fastly CDN,
+[open a support ticket](https://console.platform.sh/-/users/~/tickets/open).
+For information on how to subscribe to a managed Fastly CDN,
```

Comment:
:pick: To increase directness and potentially avoid confusion with subscriptions.
```suggestion
To add a managed Fastly CDN to your project,
```